### PR TITLE
Revisit Standard Library Dependabot

### PR DESCRIPTION
**Pull request by TharindaDilshan**
Fixes: #961 

- Update repositories even if a run fails
- Log modules that are being updated
- Avoid adding ballerinax module versions in the Distribution repo

### CODE REVIEW

File: dependabot/update_repositories.py [hunk 1]
```diff
@@ -14,229 +14,251 @@
...
+                try:
+                    commit_changes(updated_data, current_version, repo, module['name'], module['version'])
+                    create_pull_request(repo, current_version, module['name'], module['version'])
+                    print("Bump " + module['name'] + " version from " + current_version + " to " + module['version'])
+                except:
+                    continue
+            else:
+                print(module['name'] + " version is already the lastest version " + current_version)
+        print("-------------------------------------------------------------------------------------")        
             time.sleep(30)
```
Comment (ThisaruGuruge) [inline dependabot/update_repositories.py:138]: Wrong indentation?

File: release/src/version_update/update_distribution.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
+import urllib.request
```
Comment (ThisaruGuruge) [inline release/src/version_update/update_distribution.py:1]: Unused import?

File: release/src/version_update/update_distribution.py [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+import json
+import sys
+import os
+import semver
+from retry import retry
```
Comment (ThisaruGuruge) [inline release/src/version_update/update_distribution.py:6]: Unused import?

**Review by ThisaruGuruge — COMMENTED** [review]
Please check whether the `module_dependencies/update_version_data.py` file follows the naming conventions.

File: dependabot/update_repositories.py (hunk 1 above)
Comment (TharindaDilshan) [reply dependabot/update_repositories.py:137]: Fixed

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**